jnigen/example/in_app_java/src/android_utils/dartjni.h

```diff
@@ -2,8 +2,9 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-#pragma once
+#pragma once 
```

Comment:
Spurious whitespace at end of line.

(I guess we don't run a formatter for our C files on the CI that complains if files are not formatted correctly. We only do it for Dart files.)